hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/s3guard/ITestS3GuardToolLocal.java

```diff
@@ -57,13 +59,18 @@
   private static final String[] ABORT_FORCE_OPTIONS = new String[] {"-abort",
       "-force", "-verbose"};
 
+  @Override
+  protected Configuration createConfiguration() {
+    Configuration conf = super.createConfiguration();
+    conf.set(S3_METADATA_STORE_IMPL, S3GUARD_METASTORE_LOCAL);
```

Comment:
need to use the `removeBaseAndBucketOverrides()` code in S3ATestUtils to make sure that any per-bucket setting has turned it on. Look for other uses to see what to do

Reply:
Thanks. I added code to reset `fs.s3a.metadatastore.impl` and `fs.s3a.bucket.probe`.

Reply:
thanks. FWIW that removeBaseAndBucket call is varags; you can remove as many as you need in one go. Code is fine as as is though

Reply:
Sure. I removed the duplicate call.